apps/www/components/v0-button.tsx

```diff
@@ -63,7 +63,7 @@ export function V0Button({
         <Button
           aria-label="Edit in v0"
           className={cn(
-            "z-50 h-[calc(theme(spacing.7)_-_1px)] gap-1 rounded-[6px] bg-black px-3 text-xs text-white hover:bg-black hover:text-white dark:bg-white dark:text-black",
+            "z-49 h-[calc(theme(spacing.7)_-_1px)] gap-1 rounded-[6px] bg-black px-3 text-xs text-white hover:bg-black hover:text-white dark:bg-white dark:text-black",
```

Comment:
`z-49` doesn't exist.

```suggestion
            "z-40 h-[calc(theme(spacing.7)_-_1px)] gap-1 rounded-[6px] bg-black px-3 text-xs text-white hover:bg-black hover:text-white dark:bg-white dark:text-black",
```